### PR TITLE
Navigating pages

### PR DESCRIPTION
**Pull request by Easiv**
# Tasks:
- [ ] Changing pages of current book

# Added value:
- [ ] It displays content of the pages by navigating them by numbers below

### CODE REVIEW

File: app/routes/books/book/pages.js [hunk 1]
```diff
@@ -4,11 +4,13 @@ import RSVP from 'rsvp';
...
     return RSVP.hash({
-      pages: this.store.findAll('page', {
-        orderBy: 'book_id',
+      pages: this.store.query('page', {
+        orderBy: 'book',
         equalTo: this.modelFor('books.book').get('id')
       }),
       book: this.modelFor('books.book')
     });
+    console.log(this.modelFor('books.book').get('id'));
```
Comment (PoslinskiNet) [inline app/routes/books/book/pages.js]: ✂️

File: app/routes/books/book/pages.js [hunk 2]
```diff
@@ -4,11 +4,13 @@ import RSVP from 'rsvp';
...
+    console.log('tu jestem');
```
Comment (PoslinskiNet) [inline app/routes/books/book/pages.js]: ✂️

File: app/routes/books/book/pages.js [hunk 3]
```diff
@@ -4,8 +4,8 @@ import RSVP from 'rsvp';
 export default Route.extend({
   model() {
     return RSVP.hash({
-      pages: this.store.findAll('page', {
-        orderBy: 'book_id',
+      pages: this.store.query('page', {
+        orderBy: 'book',
```
Comment (PoslinskiNet) [inline app/routes/books/book/pages.js]: Why not bookId ?

File: app/controllers/books/book/pages.js [hunk 4]
```diff
@@ -45,10 +45,10 @@ export default Controller.extend({
       }
     },
     newPage() {
-      this.store.createRecord('page', {
+      this.get('store').createRecord('page', {
         number: 1,
-        content: document.querySelector('#textArea').innerText,
-        book: this.model.book,
+        content: '',
+        book: this.get('model.book'),
```
Comment (PoslinskiNet) [inline app/controllers/books/book/pages.js]: Does it work correctly with models now?

File: app/routes/books/book/pages/page/edit.js [hunk 5]
```diff
@@ -0,0 +1,7 @@
+import Route from '@ember/routing/route';
+
+export default Route.extend({
+  // model(params) {
```
Comment (PoslinskiNet) [inline app/routes/books/book/pages/page/edit.js]: ✂️  ?

File: app/templates/books/book/pages/page/edit.hbs [hunk 6]
```diff
@@ -0,0 +1,6 @@
+
```
Comment (PoslinskiNet) [inline app/templates/books/book/pages/page/edit.hbs]: ✂️ and use indentation.

File: app/routes/books/book/pages/page/edit.js (hunk 5 above)
Comment (Easiv) [reply app/routes/books/book/pages/page/edit.js]: /edit is obsolete and was deleted

File: app/templates/books/book/pages/page/edit.hbs (hunk 6 above)
Comment (Easiv) [reply app/templates/books/book/pages/page/edit.hbs]: /edit is obsolete and was deleted

Comment (PoslinskiNet) [reply app/templates/books/book/pages/page/edit.hbs]: I still can see this. Did you delete this in another PR?

Comment (Easiv) [reply app/templates/books/book/pages/page/edit.hbs]: Yes, in the page-operations

**Pull request merged**